.env.example

```
@@ -45,8 +44,7 @@ APPSMITH_MAIL_SMTP_TLS_ENABLED=
 #APPSMITH_SENTRY_ENVIRONMENT=
 
 # Configure cloud services
-# APPSMITH_CLOUD_SERVICES_BASE_URL="https://release-cs.appsmith.com"
-
+APPSMITH_CLOUD_SERVICES_BASE_URL="https://release-cs.appsmith.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)